api/controller.go

```diff
@@ -167,6 +167,7 @@ func (c *Controller) Configure(api *operations.LakefsAPI) {
 	api.ConfigGetConfigHandler = c.ConfigGetConfigHandler()
 
 	api.RefsRefsDumpHandler = c.RefsRefsDumpHandler()
+	api.RefsRefsRestoreHandler = c.RefsRefsRestoreHandler()
```

Comment:
These names are somewhat funny: by default I would expect "refs dump" to dump refs, not sure why it needs to be "refs refs dump".

Reply:
fixed